### DISCUSSION TITLE
How to close but this window?

### DISCUSSION THREAD

**Discussion by yinghaoyu** [ayamir/nvimdots#1097] [Q&A] [answered]
We can see `close but this` in vscode. e.g., I had opened 3 files by tabe, then focused on one file in current tabe and quickly closed the other 2 tabes.

**Comment by charliie-dev** [comment]
no idea what your saying. dont use analogies, give us straight examples.

also, plz dont assume everyone used vscdoe or other editor/IDE before.

**Reply by yinghaoyu** [reply]
e.g. I had opened 3 files by tabe, then focused on one file in current tabe and quickly closed the other 2 tabes.

**Comment by ayamir** [comment, marked as answer]
`:BufDelOthers`
